### PR TITLE
Use console backend to display user saved searches

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related issue: https://github.com/stolostron/backlog/issues/23869

### CODE REVIEW

File: frontend/src/resources/userpreference.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    name: username,
+                },
+                spec: {
+                    savedSearches,
+                },
+            })
+        })
+        .catch((error) => {
+            // eslint-disable-next-line no-console
+            console.error(error)
```
Comment (KevinFCormier) [inline frontend/src/resources/userpreference.tsx:54]: Did you intend to keep these?

**Review by KevinFCormier — COMMENTED** [review]
Left one question inline about the console logs. Also, please review the code smells. I am not concerned if they are in test cases, but there might be some there we should fix.

File: frontend/src/resources/userpreference.tsx (hunk 1 above)
Comment (zlayne) [reply frontend/src/resources/userpreference.tsx:54]: I was planning to include them yes. And I will comb through the code smells - thanks.

**Pull request merged**